nsxt/lb_utils.go

```diff
@@ -237,3 +240,123 @@ func resourceNsxtLbMonitorDelete(d *schema.ResourceData, m interface{}) error {
 	}
 	return nil
 }
+
+func resourceNsxtPolicyLBMonitorProfileExists(id string, connector client.Connector, isGlobalManager bool, resourceType string) (bool, error) {
```

Comment:
Is `isGlobalManager` seems redundant here, global manager is not supported.
Also my personal preference would be to define a single function for all *Exists* and *Delete* implementations for LB monitor, since `resourceType` is only used for printing errors. For printing purposes, this ca be just LbMonitor, without specifying exact type. But I don't insist :)

Reply:
Makes sense, I have removed these 2 parameters from the function